### PR TITLE
Updates for perf

### PR DESCRIPTION
**Pull request by jeffrifwald**
I've updated a few things to increase the scores in our Lighthouse report. @ananavati @donsalari

- All images should have alt tags
- Our templates need viewport tags
- Our main JS bundle tag should be at the bottom of the body

Doing these few things brought down our first meaningful paint time from 3500ms to 720ms and our time to interactive from 4500ms to 960ms. Our last goal for perf should be to further increase our perceptual speed index score. 

@aweary I'll also put in a PR to update the boilerplate with these perf goals in mind.

screen shot 2016-11-17 at 3 03 46 pm

### CODE REVIEW

File: client/components/home.jsx [hunk 1]
```diff
@@ -43,28 +43,28 @@ const styles = {
...
-        this.setState({testResult: `GET SUCCEEDED with status ${resp.status}` });
-      } else {
-        this.setState({testResult: `GET FAILED with status ${resp.status}` });
-      }
-    })
-    .catch((e) => {
-      this.setState({testResult: `GET FAILED: ${e.toString()}`});
-    });
-  }
+  //   fetch("/getBeerStyles", {
```
Comment (jeffrifwald) [inline client/components/home.jsx:49]: @donsalari I commented this out until we have the redux store going. Fetching on the server causes the component to not be server side rendered.

File: server/plugins/webapp/index.html [hunk 2]
```diff
@@ -2,14 +2,16 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     {{META_TAGS}}
     <title>{{PAGE_TITLE}}</title>
     {{PREFETCH_BUNDLES}}
-    {{WEBAPP_BUNDLES}}
+    {{WEBAPP_HEADER_BUNDLES}}
```
Comment (jeffrifwald) [inline server/plugins/webapp/index.html:9]: This bundle will contain the css and manifest files.

File: server/plugins/webapp/index.html [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
     <link href='https://fonts.googleapis.com/css?family=Gabriela|Open+Sans' rel='stylesheet' type='text/css'>
 </head>
 <body>
 <div class="js-content">{{SSR_CONTENT}}</div>
+{{WEBAPP_BODY_BUNDLES}}
```
Comment (jeffrifwald) [inline server/plugins/webapp/index.html:14]: This bundle contains the js build.

**Comment by ananavati** [comment]
interesting perf improvements, wonder how these numbers match up to https://www.webpagetest.org/ once we deploy this app.

**Pull request merged**